app/jobs/calendar_importer/event_resolver.rb

```diff
@@ -118,7 +118,7 @@ def event_override_strategy(place, address: nil)
       if place.present?
         # place = 'calendar.place'
         # address = 'calendar.place.address'
-        place = calendar.place
+        # place = calendar.place
```

Comment:
What's the reason for commenting out handling of `place` after checking whether it's present or not?